src/hash.cr

```diff
@@ -181,30 +264,66 @@ class Hash(K, V)
     self
   end
 
+  # Returns an `EntryIterator`.
```

Comment:
I think it's marked as nodoc, so it would be nice to briefly describe what it does here.